packages/docs-markdown-to-mail/src/controllers/mail-controller.ts

```diff
@@ -160,42 +170,83 @@ export function convertMarkdownToHtml() {
+	imageAsBase64 = fs.readFileSync(bannerImagePath, 'base64');
+	attachments.push({
+		'@odata.type': '#microsoft.graph.fileAttachment',
+		name: bannerImageName,
+		contentType: `image/${imageExtension}`,
+		contentBytes: imageAsBase64,
+		contentId: bannerImageName,
+		isInline: true
+	});
 
-		const titleImage = `
-      <table class="MsoTableGrid" border="0" cellspacing="0" cellpadding="0" style="background:#2E4B70;border-collapse:collapse;border:none"><tr style="height:3.65pt"><td width="443" valign="top" style="width:332.6pt;padding:0in 0in 0in .3in;height:3.65pt"><p class="MsoNormal"><span style="font-size:14.0pt;font-family:&quot;Segoe UI&quot;,sans-serif;color:#0078D4"><img width="322" height="67" style="width:3.3541in;height:.6944in" id="Picture_x0020_3" src="cid:${bannerFileName}" alt="&quot;Content + Learning team name&quot;"></span><span style="font-size:14.0pt;font-family:&quot;Segoe UI&quot;,sans-serif;color:#0078D4"><o:p></o:p></span></p></td><td width="190" rowspan="2" valign="bottom" style="width:142.5pt;padding:0in 0in 0in 0in;height:3.65pt"><p class="MsoNormal" align="right" style="text-align:right"><span style="font-size:14.0pt;font-family:&quot;Segoe UI&quot;,sans-serif;color:#0078D4"><img width="190" height="158" style="width:1.9791in;height:1.6458in" id="Picture_x0020_2" src="cid:${bannerImageName}" alt="&quot;Decorative Learn Pathways image&quot;"></span><span style="font-size:14.0pt;font-family:&quot;Segoe UI&quot;,sans-serif;color:#0078D4"><o:p></o:p></span></p></td></tr><tr style="height:10.75pt"><td width="443" valign="top" style="width:332.6pt;padding:0in 0in 0in .3in;height:10.75pt"><p class="MsoNormal"><span style="font-size:18.0pt;font-family:&quot;Segoe UI&quot;,sans-serif;color:white">${emailSubject}<o:p></o:p></span></p></td></tr></table>`;
+	let templateType: any;
+	let subjectPrefix: string;
 
-		emailBody = titleImage.concat(emailBody);
+	try {
+		let templateTypeMatch: any = metadata.match(msCustomRegex);
+		templateType = templateTypeMatch[1];
+	} catch (error) {
+		showStatusMessage(`Not a communication template.`);
+	}
+
+	const communicationDate = getCommunicationDate();
 
-		imageAsBase64 = fs.readFileSync(bannerLabelPath, 'base64');
-		attachments.push({
-			'@odata.type': '#microsoft.graph.fileAttachment',
-			name: bannerFileName,
-			contentType: `image/${imageExtension}`,
-			contentBytes: imageAsBase64,
-			contentId: bannerFileName,
-			isInline: true
-		});
-		imageAsBase64 = fs.readFileSync(bannerImagePath, 'base64');
-		attachments.push({
-			'@odata.type': '#microsoft.graph.fileAttachment',
-			name: bannerImageName,
-			contentType: `image/${imageExtension}`,
-			contentBytes: imageAsBase64,
-			contentId: bannerImageName,
-			isInline: true
-		});
+	switch (templateType) {
+		case 'docs-coming-soon':
+			try {
+				subjectPrefix = 'Coming Soon: ';
+				bannerImagePath = join(extensionPath, 'images/coming-soon.png').replace(/\\/g, '/');
+				bannerImageName = basename(bannerImagePath);
```

Comment:
Can you pull these two lines below the switch so that it's not duplicated in the other cases?